Build/libHttpClient.UnitTest.141.TE/libHttpClient.UnitTest.141.TE.vcxproj

```diff
@@ -34,7 +34,7 @@
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <AdditionalOptions>/bigobj /Zm512 %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;%(AdditionalUsingDirectories)</AdditionalUsingDirectories>
+      <AdditionalUsingDirectories>C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\lib\store\references;C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\vcpackages</AdditionalUsingDirectories>
```

Comment:
>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\vcpackages [](http://example.com/codeflow?start=163&length=88)

intentional hard coded path? is there a variable that should be used here?

Reply:
I don't see anything obvious in env vars.  Its just to enable build machine so using hardcoded path is fine as it doesn't matter if it doesn't exist on non-build machines